vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -573,7 +575,7 @@ public void onClick(DialogInterface dialog, int which) {
                         })
                         .show();
 
-                VectorApp.getInstance().clearAppCrashStatus();
+                VectorUncaughtExceptionHandler.INSTANCE.clearAppCrashStatus(this);
```

Comment:
i think this should be cleared just after the didCrash test or in a finally?

Reply:
Yes, TBH, I do not understand this try catch block, and I'm really tempted to remove it

Reply:
Should be interesting to search in (old) crash logs if `appCrashedAlert failed` occurred